Jenkinsfile

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env groovy
+@Library('conjur@cyberark-bash-lib-24') _
```

Comment:
Should not be merged while this remains, only necessary for testing for the pipeline library change merges.